pyfr/solvers/base/elements.py

```diff
@@ -383,6 +392,25 @@ def get_vect_fpts_for_inter(self, eidx, fidx):
 
         return (self._vect_fpts.mid,)*nfp, rmap, cmap, rstri
 
+    def _get_vect_upts_for_inter(self, eidx, fidx):
```

Comment:
I think we should see if we can template these boiler plate get functions now we have quite a few of them.